Codeinwp/otter-blocks
# Copy&Paste Animation + Adaptors update

Closes #1446.


### Summary


- Update adaptors for the recent changes.
- Copy & Paste animations.

----

### Test instructions


Now the animation settings should also be copied.

ℹ️ As always, the copy-paste between different block types might not behave as expected.



---- 

### Checklist before the final review
